src/plugins/dashboard/public/application/embeddable/viewport/README.md

```diff
@@ -0,0 +1,23 @@
+# How to test print-optimized reports
+
+To see the page-optimized dashboard print out use the browser's built-in print preview.
+
+**Note**: It is recommended to use Chrome or Chromium since that will use the same
+implementation as the Screenshotting plugin.
+
+1. Ensure the dashboard is rendering in "print" mode. This can be hacked by
```

Comment:
You can also get into print mode by injecting `&_a=(viewMode:PRINT)` onto the end of the URL, not sure if we should include that in these instructions!

Reply:
Yeah, I think that is even better!